### DISCUSSION TITLE
Hitcircle in Editor

### DISCUSSION THREAD

**Discussion by NympheGitHub** [ppy/osu#18344] [Q&A] [answered]
I just want to ask if it is intentional, on lazer if you open editor and look at the map, you will see circles being pure white after hitting patterns. On stable we can clearly see the numbers after the hit. On lazer this will be a little annoying to mappers while map or look at the maps. Got some feedback from mappers that already see this and they are saying that is really annoying to see. You can see on the photo here:

![](https://kiyoshi.s-ul.eu/NgQ0cdZU)

Edit: (Different skins do the same thing on lazer just want to metion it.)

**Comment by peppy** [comment, marked as answer]
Check the latest changelog for more details on this change.
